### DISCUSSION TITLE
django.core.exceptions.ImproperlyConfigured: Plugin netbox_secrets requires NetBox minimum version 4.0.0.

### DISCUSSION THREAD

**Discussion by moseisleydk** [netbox-community/netbox#17446] [Help Wanted!] [answered]
I am trying to rebuild netbox 3.7 and get:

django.core.exceptions.ImproperlyConfigured: Plugin netbox_secrets requires NetBox minimum version 4.0.0.

Problably because https://github.com/Onemind-Services-LLC/netbox-secrets has a version 2.X now.

How can I ensure a version 1.X is used? 

This is on docker and I have:

oot@netbox02:/pack/netbox-test# cat plugin_requirements.txt 
netbox-secrets
netbox-topology-views
root@netbox02:/pack/netbox-test# 

The same could (!) be a problem for netbox-topology-views

**Comment by a084ed22** [comment, marked as answer]
in your local_requirements.txt you should specify
`netbox-secrets <2.0`
or whichever version you need

**Reply by moseisleydk** [reply]
Awesome - Thank You !!!
